wgsl/extract-grammar.py

```diff
@@ -510,7 +761,321 @@ def read_spec(options):
                     scanner_i += scanner_parse[-1]  # Advance line index
         scanner_i += 1
 
-    result[scanner_rule.name()]["_comment"] = [["`/\\/\\/.*/`"]]
+    RULE_NAME_CODE_POINTS = string.ascii_lowercase + string.digits + "_"
+    bnfdialect_data = ""
+
+    # Read the contents of the bnfdialect file
+    with open(options.syntax_filename, "r") as file:
+        bnfdialect_data = file.read()
+
+    scan_in = {"name": "blank", "value": "", "content": []}  # blank, comment, rule
+
+    syntax_dict = {}
+
+    s_i = 0
+    s_v = bnfdialect_data[s_i]
+    while s_i < len(bnfdialect_data):
+        if scan_in["name"] == "blank":
+            if s_v == "/" and s_i + 1 < len(bnfdialect_data):  # Comment start
+                if bnfdialect_data[s_i + 1] == "*":
```

Comment:
I suggest only supporting // style comments.
Then you don't need to keep state between lines.